### ISSUE TITLE
http 502 (nginx)

### ISSUE THREAD

**Issue by RuralAnemone** [ThatFruedDued/blooket-utility#93]
is it just me or is the website down?
![image](https://user-images.githubusercontent.com/64103005/193474860-44094413-25d4-448d-94e3-520cffadbfae.png)

**Comment by VillainsRule4000** [comment]
It's down.

**Comment by RuralAnemone** [comment]
cool

**Comment by RuralAnemone** [comment]
thanks

**Comment by ThatFruedDued** [comment]
This issue is still valid; please do not close it.

**Comment by RuralAnemone** [comment]
oop sorry lol :)

**Comment by VillainsRule4000** [comment]
When can we expect a fix?

**Comment by ThatFruedDued** [comment]
There's an underlying issue with the Blooket Utility that needs to be addressed before I can fix this. I really don't know when that is going to happen, sorry.

**Comment by Wilke000** [comment]
Yea it's down for me too

**Comment by Aman122608** [comment]
> There's an underlying issue with the Blooket Utility that needs to be addressed before I can fix this. I really don't know when that is going to happen, sorry.

Any updates?

**Comment by ThatFruedDued** [comment]
Yes, I am currently working on a new version of the Blooket Utility (v3) that will overhaul the backend of how it works, making it able to work again on the latest version of Blooket.

**Comment by timmytheman123** [comment]
privacy error
![image](https://user-images.githubusercontent.com/91916360/196173553-0aef69c4-c644-4c54-a5c5-b8570bca1d5b.png)

**Comment by Aman122608** [comment]
> Yes, I am currently working on a new version of the Blooket Utility (v3) that will overhaul the backend of how it works, making it able to work again on the latest version of Blooket.

Any update now?

**Comment by BannanaPants** [comment]
Maybe when you publish it, if it's not too hard, use a different host because for my school district that host site is blocked. And not just yours, all of the sites that have that host site are blocked.

**Comment by bloohacked** [comment]
> Maybe when you publish it, if it's not too hard, use a different host because for my school district that host site is blocked. And not just yours, all of the sites that have that host site are blocked.

well it's down for everyone not just your school district

**Comment by Jesus-boop-collab** [comment]
> 

so im gonna assume you abandoned this project/

**Comment by ThatFruedDued** [comment]
Read the README for more information.
